### DISCUSSION TITLE
Help please!

### DISCUSSION THREAD

**Discussion by HipsterJesus115** [bennycode/coinbase-pro-node#499] [Q&A]
So I am stuck just trying to figure out how to just get the rest example given to work. I have it installed properly using node but every time I try to use it there's some sort of stupid error. 

First trying to use it I got the error "Cannot use import statement outside of a module"
So I said okay and added the "type=module" to my code.

Then I get "Failed to resolve module specifier "CoinbasePro". Relative references must start with either "/", "./", or "../""
So I tried adding one of the characters it says. 

Regardless of which one I use I get the error "net::ERR_ABORTED 404 (Not Found)"

The file I am running this from is in the root directory of the site. Does it have to be within the node_modules folder? 

If anyone can assist me id appreciate it please!

**Comment by bennycode** [comment]
Hi @HipsterJesus115, are you trying to run "coinbase-pro-node" from a cloned local repository or have you added it as a dependency to your own project?

I have linked read world examples from the README. By having a look at those example you will see how to make use of "coinbase-pro-node": https://github.com/bennycode/coinbase-pro-node/tree/v3.0.4#real-world-examples
